target/linux/x86/patches-5.10/011-tune_lzma_options.patch

```diff
@@ -0,0 +1,26 @@
+Index: linux-5.10.16/scripts/Makefile.lib
+===================================================================
+--- linux-5.10.16.orig/scripts/Makefile.lib
++++ linux-5.10.16/scripts/Makefile.lib
+@@ -370,7 +370,7 @@ quiet_cmd_bzip2 = BZIP2   $@
+ # ---------------------------------------------------------------------------
+ 
+ quiet_cmd_lzma = LZMA    $@
+-      cmd_lzma = { cat $(real-prereqs) | $(LZMA) e -d20 -lc1 -lp2 -pb2 -eos -si -so; $(size_append); } > $@
++      cmd_lzma = { cat $(real-prereqs) | $(LZMA) e -lc8 -eos -si -so; $(size_append); } > $@
+ 
+ quiet_cmd_lzo = LZO     $@
+       cmd_lzo = { cat $(real-prereqs) | $(KLZOP) -9; $(size_append); } > $@
+Index: linux-5.10.16/arch/x86/include/asm/boot.h
+===================================================================
```

Comment:
please remove these quilt edit index references to be consistent with other patches

Reply:
see above ...